EasyDisenchant.lua

```diff
@@ -364,7 +405,20 @@ end
 _M.OnLoad = function(self)
 	-- Register command.
 	SLASH_DISENCHANT1, SLASH_DISENCHANT2 = "/disenchant", "/de";
-	SlashCmdList["DISENCHANT"] = self.InvokeWindowOpen;
+	SlashCmdList["DISENCHANT"] = 
+    function(msg)
+        -- if command is /disenchant reset or /de reset; reset the blacklist.
+        if(strlower(msg) == "reset") then
```

Comment:
When performing string operations multiple times, such as `strlower` here, the output should be stored in a variable rather than the call being executed twice. Basic optimization.